spec/models/in_person_enrollment_spec.rb

```diff
@@ -103,4 +103,59 @@
       end
     end
   end
+
+  describe 'minutes_since_established' do
+    let(:enrollment) {
+      create(
+        :in_person_enrollment, :passed, enrollment_established_at: Time.zone.now - 2.hours
+      )
+    }
+
+    it 'returns number of minutes since enrollment was established' do
+      expect(enrollment.minutes_since_established).to be_within(0.01).of(120)
```

Comment:
A `be_within` seems prone to test flakiness. Should we use [`freeze_time`](https://api.rubyonrails.org/v5.2.6/classes/ActiveSupport/Testing/TimeHelpers.html) to avoid dealing with the time delta, similar to what we do in other specs?

Reply:
Yeah good point. I was worried it would break something because of how much `Time.zone.now` is used but I dove in and nothing broke so it's done in 7e50a95069563b6dd7558d1bf8633a9393d21b1e